contracts/factories/BridgeFactory.sol

```diff
@@ -142,24 +150,27 @@ contract BridgeFactory is IERC1155TokenReceiver, TieredOwnable {
 
     } else {
       revert("BridgeFactory#onERC1155Received: INVALID_TOKEN");
-    }
-
+    }  
+    
+    emit Deposit(_from, _data.readBytes32(0));
```

Comment:
Any reason for not using `abi.decode(_data, (bytes32))`?

Reply:
Not really, could use that indeed.